data/avro-schema/src/main/resources/avro/net/corda/data/permissions/management/PermissionManagementRequest.avsc

```diff
@@ -24,6 +24,11 @@
         "net.corda.data.permissions.management.user.AddRoleToUserRequest",
         "net.corda.data.permissions.management.user.RemoveRoleFromUserRequest",
         "net.corda.data.permissions.management.user.DeleteUserRequest",
+        "net.corda.data.permissions.management.group.AddRoleToGroupRequest",
+        "net.corda.data.permissions.management.group.ChangeGroupParentIdRequest",
```

Comment:
While we are on it, shall we also introduce: `ChangeUserParentGroupIdRequest` ?

Reply:
Ah, good call. Added.